### ISSUE TITLE
`module` in `package.json`

### ISSUE THREAD

**Issue by brettz9** [localForage/localForage#805]
With your source in ES6 modules, how about adding `module` to `package.json` for ready discovery by bundlers?

https://stackoverflow.com/a/42817320/271577

Thanks!

**Comment by tofumatt** [comment]
Yeah, that's a fine idea to me! I'd happily accept a patch that did this 😄

**Comment by tofumatt** [comment]
Funny enough, this broke addons-frontend build https://travis-ci.org/mozilla/addons-frontend/jobs/360199156

I'm going to make sure this isn't too bad of a breaking change.

**Comment by thgreasi** [comment]
@tofumatt it might be that we have to also include the src folder to the published npm package!

**Comment by thgreasi** [comment]
Oh, forget that, we already do include them...

**Comment by brettz9** [comment]
While I'd expect that that package (and any consumers) should be able to work with `module`, taking into account as they should, of the same compiling requirements of `localForage` (unless they are, less efficiently, targeting one of your pre-made bundles), for those not wishing to go to the trouble, this should, I believe, be addressable for such present webpack consumers in the meantime by simply adding `mainFields` without `module` to `resolve`:

https://webpack.js.org/configuration/resolve/#resolve-mainfields

```json
"mainFields": ["browser", "main"]
```

**Comment by tofumatt** [comment]
Ah, we target node so perhaps that's the issue. I'll update the release notes to include information about how to easily update a webpack config, hopefully there aren't too many issues. Thanks! 👍

**Comment by tofumatt** [comment]
This is breaking things for other users too and I'd rather test it out more thoroughly and get things working better out of the box. I'm going to re-open this and revert the change in 1.7.1, but I'm open to this idea. We just have to make sure webpack will build `src/localforage.js` out-of-the-box better than it does now.

**Comment by tofumatt** [comment]
I think you're right; though the issue with consuming localForage is seemingly around template strings and I'm surprised it failed as we use them in add-ons front-end as well.

To be clear I would happily fix this bug but I'd like to explore more why the build failures are happening before rushing out the release. But I do think this is a good idea 😄

**Comment by brettz9** [comment]
Could the template issue relate to not running babel during or at the right part of the rollup routine?

**Comment by tofumatt** [comment]
I think so, but because it happened to me and another developer on a different project I’m guessing this is a common configuration issue. It would at least be nice to know which kind of configuration change we should be recommending over just telling people to continue to consume the `dist/` version if they get errors ;-)
